LeaderboardBackend/Models/Entities/Timestamps.cs

```diff
@@ -0,0 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+using NodaTime;
+
+/// <summary>
+///     Base class that provides timestamp fields to all Entities.
+/// </summary>
+public class Timestamps
```

Comment:
I don't think this class is meant to be here anymore.

Reply:
Eh

Oops.